README.md

```diff
@@ -38,7 +38,7 @@ Jest matcher that performs image comparisons using [pixelmatch](https://github.c
 * `customDiffConfig`: Custom config passed [pixelmatch](https://github.com/mapbox/pixelmatch#pixelmatchimg1-img2-output-width-height-options) (See options section)
   * By default we have set the `threshold` to 0.01, you can increase that value by passing a customDiffConfig as demonstrated below.
   * Please note the `threshold` set in the `customDiffConfig` is the per pixel sensitivity threshold. For example with a source pixel colour of `#ffffff` (white) and a comparison pixel colour of `#fcfcfc` (really light grey) if you set the threshold to 0 then it would trigger a failure *on that pixel*. However if you were to use say 0.5 then it wouldn't, the colour difference would need to be much more extreme to trigger a failure on that pixel, say `#000000` (black)
-* `customSnapshotsDir`: (default: `__image_snapshots__`) A custom directory to keep this snapshot in
+* `customSnapshotsDir`: (default: `__image_snapshots__`) A custom absolute path of a directory to keep this snapshot in
```

Comment:
It's not an absolute path though right? It's a path relative to the project root. Also, @anescobar1991 that's not the appropriate default value since it is relative to the test.

Reply:
No it is an absolute path https://github.com/americanexpress/jest-image-snapshot/blob/master/src/index.js#L53 but yes the default should change, I can do that in a separate PR.